data-source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -54,4 +54,64 @@
       },
     }],
   },
+  firstPersonPlaceholder: {
+    placeholder: 'first_person_name',
+    transforms: [{
+      transform: 'concatenate_list',
+      arguments: {
+        list_to_concatenate: {
+          source: 'answers',
+          identifier: ['first-name', 'last-name'],
+          list_item_selector: {
+            source: 'location',
+            id: 'list_item_id'
+          }
+        },
+        delimiter: ' '
+      }
+    }]
+  },
+  secondPersonPlaceholder: {
+    placeholder: 'second_person_name',
+    transforms: [{
+      transform: 'concatenate_list',
+      arguments: {
+        list_to_concatenate: {
+          source: 'answers',
+          identifier: ['first-name', 'last-name'],
+          list_item_selector: {
+            source: 'location',
+            id: 'to_list_item_id'
+          }
+        },
+        delimiter: ' '
+      }
+    }]
+  },
+  firstPersonNamePossessivePlaceholder: {
+    placeholder: 'first_person_name_possessive',
+    transforms: [{
+        transform: 'concatenate_list',
+        arguments: {
+          list_to_concatenate: {
+            source: 'answers',
+            identifier: ['first-name', 'last-name'],
+            list_item_selector: {
+              source: 'location',
+              id: 'list_item_id'
+            }
+          },
+          delimiter: ' '
+        }
+      },
+      {
+        transform: 'format_possessive',
+        arguments: {
+          string_to_format: {
+            source: 'previous_transform'
+          }
+        }
+      }
+    ]
+  },
```

Comment:
These placeholder shouldn't be in this file. 
- This file is intended for placeholders that are used in a number of places
- They don't make sense outside of the context of the relationship collector block

Reply:
Fair enough, I originally had them inside the relationships file. I'll move them back